src/atoms/feedCardItem/feedCardItem.tsx

```diff
@@ -0,0 +1,146 @@
+  const token0Usd = `($${token0Amount.toFixed(2)})`;
+  const token1Usd = `($${token1Amount.toFixed(2)})`;
+  // TODO: Add all tx types here
+  const txTypePreposition = txType === 'swap' ? 'to' : 'text';
+  return (
+    <MasterContainer
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      key={hash}
+      isMulti={isMulti}
+    >
+      <TxTypeWrapper>
+        <IconContainer>
+          <ChainIcon><IconWrapper icon={chainIcons[chain]} height="14px" width="14px" /></ChainIcon>
+          <DexIcon><IconWrapper icon={dexIcons[dex]} height="28px" width="28px" /></DexIcon>
+        </IconContainer>
+        <TxTypeContainer>
+          <Text size="S-Regular">{txType}</Text>
+          <Text size="S-Regular" color={theme.textShades.SHADE_MINUS_2}>{dex}</Text>
+        </TxTypeContainer>
+      </TxTypeWrapper>
+      <CenterContentContainer>
+        <XPartyContent>
+          <Text size="S-Bold" color={theme.colors.primary.UWL_BLUE}>{Number(token0Amount).toFixed(4)}</Text>
+          <HintsAndHovers
+            id={`${token0Address}${hash}`}
+            place="top"
+            hint={token0Symbol}
+            icon={(
+              <TokenIcon
+                baseUrl={LogoUrlBase}
+                tokenAddress={token0Address.toLowerCase()}
+              />
+            )}
+          />
+          <Text size="S-Regular" color={theme.textShades.SHADE_MINUS_2}>{token0Usd}</Text>
+        </XPartyContent>
+        <Text size="S-Regular" color={theme.textShades.SHADE_MINUS_3}>{txTypePreposition}</Text>
+        <YPartyContent>
+          <Text size="S-Bold" color={theme.colors.primary.UWL_BLUE}>{Number(token1Amount).toFixed(4)}</Text>
+          <HintsAndHovers
+            id={`${token1Address}${hash}`}
+            place="top"
+            hint={token1Symbol}
+            icon={(
+              <TokenIcon
+                baseUrl={LogoUrlBase}
+                tokenAddress={token1Address.toLowerCase()}
+              />
+            )}
+          />
+          <Text size="S-Regular" color={theme.textShades.SHADE_MINUS_2}>{token1Usd}</Text>
+        </YPartyContent>
+      </CenterContentContainer>
+      <StyledLink
+        href={`https://etherscan.io/tx/${hash}`}
+        rel="noreferrer"
+        target="_blank"
+      >
+        <LinkIcon />
+      </StyledLink>
+      <SecondaryActionContainer>
+        {hover && !isMulti && (
+          <HoverItemsContainer>
+            <FeedPageMeatballMenu
+              goToItem={() => console.log('go to item')}
+              share={() => navigator.clipboard.writeText(`https://etherscan.io/tx/${hash}`)}
+              openSeaLink={`https://etherscan.io/tx/${hash}`}
+            />
+            <IconWrapper cursor="pointer" icon={<TwitterColor />} />
+          </HoverItemsContainer>
+        )}
+        {hover && isMulti && !isFirst && (
+          <HoverItemsContainer>
+            <FeedPageMeatballMenu
+              goToItem={() => console.log('go to item')}
```

Comment:
is this missing functionality?

Reply:
Good spot, not 100% sure what we are gonna have there so just gonna drop the etherscan link in for now, as I think this is again meant to be for NFTs